src/main/scala/uk/gov/nationalarchives/notifications/LambdaRunner.scala

```diff
@@ -30,7 +30,7 @@ object LambdaRunner extends App {
        |    "Records": [
        |        {
        |            "Sns": {
-       |                "Message": "{\\"success\\":true,\\"consignmentId\\":\\"fc2b60ba-8078-4e7a-b91b-a5b6c616a9bc\\", \\"environment\\": \\"some-environment\\", \\"successDetails\\":{\\"userId\\": \\"fc2b60ba-8078-4e7a-b91b-a5b6c616a9bc\\",\\"consignmentReference\\": \\"some-consignment-reference\\",\\"transferringBodyCode\\": \\"some-transferring-body-code\\"}}"
+       |                "Message": "{\\"success\\":true,\\"consignmentId\\":\\"fc2b60ba-8078-4e7a-b91b-a5b6c616a9bc\\", \\"environment\\": \\"some-environment\\", \\"successDetails\\":{\\"userId\\": \\"fc2b60ba-8078-4e7a-b91b-a5b6c616a9bc\\",\\"consignmentReference\\": \\"some-consignment-reference\\",\\"transferringBodyName\\": \\"some-transferring-body-code\\"}}"
```

Comment:
I would rename the parameter value to: "some-transferring-body-name" so that it is consistent with the parameter name change.